src/pages/ContactUs.tsx

```diff
@@ -1,10 +1,19 @@
-import icon from '../images/icon.svg'
 import BasicPage from 'src/components/BasicPage'
+import contactEnvelope from '../images/contact-envelope.svg'
+// import contactStripes from '..images/contact-stripes.svg'
 
 export default function ContactUs() {
   return (
-    <BasicPage title={'Contact Us'} icon={icon}>
-      <p>Contact Us</p>
+    <BasicPage title={'Contact Us'} icon={contactEnvelope}>
+      <p className="mb-3">Do you have questions or suggestions?</p>
+      <div className="mb-3">
+        Email:
```

Comment:
Give it a little margin to the right:
```suggestion
        <span className="me-1">Email:</span>
```